gtest/src/manager.rs

```diff
@@ -294,4 +294,75 @@ impl ExtManager {
 
         Ok(message)
     }
+
+    pub(crate) fn send_reply_impl(
```

Comment:
I'd move that to `state::mailbox::actors`, as it all related to the mailbox logic

Reply:
I think `fn send_reply_impl` should be inside `ExtManager` to maintain a clear separation of abstraction levels.

Moved `send_reply` and `claim_value` to the `manager::exec` module, where `send` extrinsic is located.

Reply:
Fixed